[2025-01-20T12:06:06.041+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-20T12:06:06.051+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_cr_table_snowflake.create_table_query manual__2025-01-20T12:06:03.660462+00:00 [queued]>
[2025-01-20T12:06:06.056+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_cr_table_snowflake.create_table_query manual__2025-01-20T12:06:03.660462+00:00 [queued]>
[2025-01-20T12:06:06.057+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-20T12:06:06.066+0000] {taskinstance.py:2889} INFO - Executing <Task(SnowflakeOperator): create_table_query> on 2025-01-20 12:06:03.660462+00:00
[2025-01-20T12:06:06.073+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1415) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-20T12:06:06.076+0000] {standard_task_runner.py:72} INFO - Started process 1417 to run task
[2025-01-20T12:06:06.078+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_cr_table_snowflake', 'create_table_query', 'manual__2025-01-20T12:06:03.660462+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/dag_create_table.py', '--cfg-path', '/tmp/tmpqk1gmzzj']
[2025-01-20T12:06:06.081+0000] {standard_task_runner.py:105} INFO - Job 10: Subtask create_table_query
[2025-01-20T12:06:06.129+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_cr_table_snowflake.create_table_query manual__2025-01-20T12:06:03.660462+00:00 [running]> on host fd6029332267
[2025-01-20T12:06:06.192+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_cr_table_snowflake' AIRFLOW_CTX_TASK_ID='create_table_query' AIRFLOW_CTX_EXECUTION_DATE='2025-01-20T12:06:03.660462+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-20T12:06:03.660462+00:00'
[2025-01-20T12:06:06.194+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-20T12:06:06.196+0000] {sql.py:278} INFO - Executing: CREATE TABLE IF NOT EXISTS employees (
id integer,
name varchar(50),
role varchar(50)
);
[2025-01-20T12:06:06.204+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-01-20T12:06:06.212+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-01-20T12:06:06.213+0000] {connection.py:414} INFO - Snowflake Connector for Python Version: 3.12.4, Python Version: 3.12.8, Platform: Linux-6.12.9-orbstack-00297-gaa9b46293ea3-aarch64-with-glibc2.36
[2025-01-20T12:06:06.214+0000] {connection.py:1197} INFO - Connecting to GLOBAL Snowflake domain
[2025-01-20T12:06:06.214+0000] {connection.py:1278} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-01-20T12:06:07.223+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2025-01-20T12:06:07.224+0000] {sql.py:553} INFO - Running statement: CREATE TABLE IF NOT EXISTS employees (
id integer,
name varchar(50),
role varchar(50)
);, parameters: None
[2025-01-20T12:06:07.333+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2025-01-20T12:06:07.333+0000] {sql.py:562} INFO - Rows affected: 1
[2025-01-20T12:06:07.334+0000] {snowflake.py:447} INFO - Rows affected: 1
[2025-01-20T12:06:07.334+0000] {snowflake.py:448} INFO - Snowflake query id: 01b9d656-3201-637f-000b-894e0001e016
[2025-01-20T12:06:07.412+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2025-01-20T12:06:07.412+0000] {connection.py:789} INFO - closed
[2025-01-20T12:06:07.449+0000] {connection.py:795} INFO - No async queries seem to be running, deleting session
[2025-01-20T12:06:07.514+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-20T12:06:07.516+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_cr_table_snowflake, task_id=create_table_query, run_id=manual__2025-01-20T12:06:03.660462+00:00, execution_date=20250120T120603, start_date=20250120T120606, end_date=20250120T120607
[2025-01-20T12:06:07.549+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-20T12:06:07.611+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-20T12:06:07.616+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
